packages/components/src/SVG/README.mdx

```diff
@@ -0,0 +1,82 @@
+---
```

Comment:
Oh, you can kill this file. It never sees the light of day since it was there for the site :)